tools/gnc_visualizer/scripts/visualizer.py

```diff
@@ -420,7 +429,7 @@ def eventFilter(self, source, event):
                         self.menuBar().hide()
             elif event.type() == QtCore.QEvent.Leave and source is self:
                 self.menuBar().hide()
-        return QtGui.QMainWindow.eventFilter(self, source, event)
+        return super(Visualizer, self).eventFilter(source, event)
```

Comment:
```suggestion
        return super().eventFilter(source, event)
```
Should be equivalent in Python 3.